analyzer/src/main/kotlin/YamlFilePackageCurationProvider.kt

```diff
@@ -35,5 +38,21 @@ class YamlFilePackageCurationProvider(
         curationFile.readValue<List<PackageCuration>>()
     }
 
-    override fun getCurationsFor(identifier: Identifier) = packageCurations.filter { it.id.matches(identifier) }
+    override fun getCurationsFor(packageId: Identifier): List<PackageCuration> {
+        val packageVersion = try {
```

Comment:
I find the name `packageVersion` confusing, especially because it can be `null`. What about `ivyVersion`?

Reply:
Ivy version of what? I wanted to have something that complements `resolvedVersion`... I'll probably go for "packageIvyVersion" and "resolvedCurationVersion".